lib/miq_automation_engine/service_models/miq_ae_service_generic_object.rb

```diff
@@ -0,0 +1,11 @@
+module MiqAeMethodService
+  class MiqAeServiceGenericObject < MiqAeServiceModelBase
+    def method_missing(method_name, *args)
+      @object.send(method_name, *args)
+    end
+
+    def respond_to_missing?(method_name, include_private = false)
```

Comment:
Thinking more about this it seems wrong to allow the callers to check private methods from automate.  We should probably drop the `include_private` parameter from both methods and use the default of `false`.